Update x12-edi-connector.adoc (#613)
mulesoft/docs-connectors: modules/ROOT/pages/x12-edi/x12-edi-connector.adoc

X12 EDI Connector Version 2.1.1

The Anypoint X12 EDI Connector lets you convert X12 messages to and from DataWeave-compatible representations using lists and maps.

> We supply the schema definitions for all https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-x12[supported X12 standard transaction sets].

Reviewer: ```suggestion
The schema definitions for all https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-x12[supported X12 standard transaction sets] are provided.
```

> Many HIPAA document versions are also supported, as listed separately in the https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-hipaa[Connector HIPAA reference page].

Reviewer: This should be an xref rather than web link

Reviewer 2: Next release for sure!

=== About the EDI Schema Language

> The X12 EDI Module uses a YAML format called ESL (for EDI Schema Language) to represent EDI schemas. Basic ESLs define the structure of EDI messages in terms of structures (transaction sets, in X12 terminology), groups (loops, in X12), segments, composites, and elements. ESLs for the all https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-x12[supported X12 versions] are included. A set of https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-hipaa[HIPAA variations] of base X12 transaction sets are also included.

Reviewer: ```suggestion
The X12 EDI Module uses a YAML format called ESL (EDI Schema Language) to represent EDI schemas. Basic ESLs define the structure of EDI messages in terms of structures (transaction sets in X12 terminology), groups (loops in X12), segments, composites, and elements. 

ESLs for all https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-x12[supported X12 versions] are included. A set of https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-hipaa[HIPAA variations] of base X12 transaction sets are also included.
```

Reviewer: Those links should be xrefs. But as long as they work right now, you can update in the next iteration when you have more time for link testing, etc.

The general tab lets you configure settings for reading and writing X12 messages:

* X12 form and validation (standard X12, HIPAA SNIP Type 1, or HIPAA SNIP Type 2).

> * Manually create or edit the list of schema definitions for the message structures to be used (which must use the same form, either X12 or HIPAA). If the list of schema definitions includes a 997 or 999 Functional Acknowledgment schema then that schema will be used to generate Functional Acknowledgments. Note - In order to be used, the Functional Acknowledgment schema type (997 or 999) must match the "Generate 999 Functional Acknowledgments" setting on the Parser tab (false for 997, true for 999).

Reviewer: ```suggestion
* Manually create or edit the list of schema definitions for the message structures to use. They must all use the same form (either X12 or HIPAA). If the list of schema definitions includes a 997 or 999 Functional Acknowledgment schema then that schema will be used to generate Functional Acknowledgments. **Note:** To use the Functional Acknowledgment schema type (997 or 999) it must match the **Generate 999 Functional Acknowledgments** setting on the Parser tab (`false` for 997, `true` for 999).
```

standard X12 schema definitions provided with the connector. However, in this case you won't see any metadata to define the structure of the
data passed in and out of the connector.

> If the schema definition configuration includes a 997 or 999 Functional Acknowledgmet schema then that schema will be used to generate Functional Acknowledgments. Note - In order to be used, the Functional Acknowledgment schema type (997 or 999) must match the generate999Acks Parser parameter (false for 997, true for 999).

Reviewer: ```suggestion
If the schema definition configuration includes a 997 or 999 Functional Acknowledgmet schema then that schema will be used to generate Functional Acknowledgments. **Note:** To use the Functional Acknowledgment schema type (997 or 999) it must match the `generate999Acks` Parser parameter (`false` for 997, true for `999`).
```
